Scheduler_Enhancement_Design.md

```diff
@@ -0,0 +1,115 @@
+# Scheduler enhancement design
+
+Status: Pending
+
+Version: Alpha
+
+Author(s): PengYiOfOpenSDS
+
+## Summary
+
+This proposal is drafted to enhance the scheduler of OpenSDS.
+
+## Motivation
+
+In the current version of OpenSDS, the user can set "extras" in the profile to select a pool when creating the volume. However, the current method of filtering the pool cannot meet the complex needs of users, so OpenSDS need to enhance the filtering method.
+
+### Goals
+
+* Design a new framework for filtering pools.
+
+### Non-Goals
+
+The proposal only satisfies the data model of the pool and profile in the current OpenSDS. If the data model of the pool or profile has changed in the future, OpenSDS need to modify the proposal again.
+
+## Proposal
+
+This proposal is drafted to enhance the scheduler of OpenSDS.
+
+### Data model impact
+
+None
+
+### REST API impact
+
+None
+
+### Security impact
+
+None
+
+### Other end user impact
+
+None
+
+### Performance impact
+
+None
+
+### Other deployer impact
+
+None
+
+### Developer impact
+
+None
+
+## Use Cases
+
+If the user creates a volume and requires the volume's AvailabilityZone to be az1, then the user can set the extras["availabilityZone"]" in the profile to "az1" or "s== az1" before creating the volume.
+
+## Implementation
+
```

Comment:
Please move the content under "##Implementation" to "##Proposal". 
Then you can remove "##Implementation".